game/scripts/vscripts/Sections/Chapter4/SectionCommunication.ts

```diff
@@ -65,16 +65,44 @@ function onStart(complete: () => void) {
                 luna.FaceTowards(playerHero.GetAbsOrigin());
             }),
 
-            tg.setCameraTarget(playerHero),
+            tg.textDialog(LocalizationKey.Script_4_Communication_1, ctx => ctx[lunaName], 3),
+            tg.setCameraTarget(undefined),
+            tg.audioDialog(LocalizationKey.Script_4_Communication_2, LocalizationKey.Script_4_Communication_2, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
+            tg.audioDialog(LocalizationKey.Script_4_Communication_3, LocalizationKey.Script_4_Communication_3, ctx => ctx[CustomNpcKeys.SlacksMudGolem]),
 
             tg.immediate(_ => goalPressVoiceChatButton.start()),
-            tg.waitForCommand(37), // Team voice
+            tg.forkAny([
+                tg.seq([
```

Comment:
What happens if you press the Team Speak button at the 6th second, which means the audio dialog won't complete before the wait's over? I would assume it would interrupt the audio dialog (haven't tested)

I would suggest using a variable, and a second sequence that waits 8 seconds and then checks that variable. If they haven't used the team speak, you move on; if they did, the second sequence does not complete (the audio one should be the one completing it).

Reply:
Fixed with completeOnCheck on boolean.